Automattic/wp-calypso
# Remove isolated block editor

#### Changes proposed in this Pull Request
The isolated block editor package has given us significant trouble with WordPress monorepo updates over the past several months. This time, it's completely blocking the release of Gutenberg to dotcom for an interesting chain of issues stemming from the removal of an experimental method in Gutenberg.

It's only used for an experiment @sarayourfriend did a year or two ago with Gutenberg in the Reader. We have no plans to actively continue work on that experiment, so it's fair to remove the code completely at this point. This lets us remove the isolated block editor package from our dependencies and that should unblock the gutenberg release.

Related discussion: p1653583842699789-slack-CBTN58FTJ

#### Testing instructions



* In calypso.live, verify that comments in the Reader work properly.



Related to https://github.com/Automattic/wp-calypso/pull/64093
